zerver/tests/test_subs.py

```diff
@@ -1309,6 +1309,19 @@ def test_subscriptions_exist_invalid_name(self):
                                   {"stream": invalid_stream_name})
         self.assert_json_error(result, "Invalid characters in stream name")
 
+    def test_existing_subscriptions_autosubscription(self):
+        # type: () -> None
+        """
+        Calling /json/subscriptions/exist on an existing stream and autosubscribe to it.
+        """
+        stream_name = self.streams[0]
+        result = self.client.post("/json/subscriptions/exists",
+                                  {"stream": stream_name, "autosubscribe": True})
+        self.assert_json_success(result)
+        json = ujson.loads(result.content)
+        self.assertIn("exists", json)
+        self.assertTrue(json["exists"])
```

Comment:
Can you make the commit message for this commit be simply:  `Add test_existing_subscriptions_autosubscription()`

Also, a very minor nit, I would say "Call" instead of "Calling" in the doc string.